.pipelines/js-ci.yml

```diff
@@ -0,0 +1,104 @@
+name: $(date:yyMM).$(date:dd)$(rev:rrr)
+
+pr:
+  branches:
+    include:
+    - main
+  paths:
+    include:
+    - source/nodejs
+
+pool:
+  name: Azure Pipelines
+  vmImage: ubuntu-20.04
```

Comment:
>ubuntu-20.04 [](http://example.com/codeflow?start=11&length=12)

is there some way to keep the pipeline running in windows hosts?

Reply:
why do you ask?
I guess I only need to change the  vmImage to windows.
-bash should be supported in Windows too.

Reply:
tests use the edge driver, we may have to update the driver used if no edge is installed in ubuntu vm